kong/tracing/instrumentation.lua

```diff
@@ -219,7 +217,7 @@ function _M.request(ctx)
     http_flavor = string.format("%.1f", http_flavor)
   end
 
-  local active_span = tracer.start_span(span_name, {
+  local active_span = tracer.start_span("kong.request", {
```

Comment:
let's rename to `kong` as discussed above
```suggestion
  local active_span = tracer.start_span("kong", {
```